zerver/lib/thumbnail.py

```diff
@@ -21,6 +26,83 @@
 # Reject uploaded emoji which are larger than this
 MAX_EMOJI_GIF_FILE_SIZE_BYTES = 128 * 1024 * 1024  # 128 kb
 
+# These are the image content-types which the server supports parsing
+# and thumbnailing; these do not need to supported on all browsers,
+# since we will the serving thumbnailed versions of them.
+THUMBNAIL_ACCEPT_IMAGE_TYPES = frozenset(
+    [
+        "image/avif",
+        "image/gif",
+        "image/heic",
+        "image/jpeg",
+        "image/png",
+        "image/svg+xml",
+        "image/webp",
+    ]
+)
```

Comment:
We probably should put in some comments documenting why these are a different set from `INLINE_MIME_TYPES`?

Reply:
Does the comment above not suffice?  I can spell it out more explicitly, but I thought "do not need to supported on all browsers, since we will the serving thumbnailed versions of them" was pretty clear.

Reply:
I was thinking comments inline in the list of formats with notes on why it's correct to include it in which list; it's always a pain to track down why we decided what we did around these format details, so I was thinking it might be a good opportunity for the code to self-document that.